web/gridt/tests/integration/resources/test_leader.py

```diff
@@ -78,19 +91,14 @@ def test_swap_leader_movement_not_subscribed(self):
 
     def test_swap_leader_not_leader(self):
```

Comment:
I'm not sure what this is supposed to test. Can you explain it to me?

Reply:
It is supposed to test that a user cannot request to swap a leader that they are not following.